ember/app/components/top-nav/template.hbs

```diff
@@ -10,7 +10,12 @@
     </div>
     <div class='uk-navbar-right'>
       <ul class='uk-navbar-nav uk-navbar-left'>
-        <div class='uk-navbar-left'></div>
+        <div class='uk-navbar-left'>
+          <div class="button-nav">          
+            <LinkTo class="uk-button uk-button-default" @route="projects">Overview</LinkTo>
+            <LinkTo class="uk-button uk-button-default" @route="projects.add">Add</LinkTo>
+          </div>
+        </div>
       </ul>
```

Comment:
That's not valid HTML. A `<ul>` must have `<li>` as children. Also, why `uk-navbar-left` inside of `uk-navbar-right`?